cli/src/pcluster/commands.py

```diff
@@ -506,6 +506,14 @@ def delete(args):
                     ).ljust(80)
                     sys.stdout.write("\r%s" % resource_status)
                     sys.stdout.flush()
+                elif isinstance(result, ApiFailure):
+                    # If stack is already deleted
+                    if f"Cluster {args.cluster_name} does not exist." in result.message:
+                        LOGGER.info("\nCluster deleted successfully.")
```

Comment:
Let's do `LOGGER.warning(f"\nCluster {args.cluster_name}
 has already been deleted or does not exist.")`